spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/TaskCommands.java

```diff
@@ -201,7 +202,10 @@ public String launch(
 					ARGUMENTS_OPTION }, help = "the commandline arguments for this launch") String arguments,
 			@CliOption(key = {
 					PLATFORM_NAME}, help = "the platform name to use for this launch",
-					unspecifiedDefaultValue = "default") String platformName)
+					unspecifiedDefaultValue = "default") String platformName,
+			@CliOption(key = {
+					ALTERNATE_CTR_APP}, help = "alternate Composed Task Runner app to use for this launch") String alternateComposedTaskRunnerApp)
```

Comment:
Can we give this a shorter name, if we are specifying it, it is implicitly an 'alternative' name, perhaps just `--composedTaskRunnerName` ?